src/main/java/org/sonarqube/gradle/SonarQubePlugin.java

```diff
@@ -99,6 +100,17 @@ private void configureTask(SonarQubeTask sonarQubeTask, Project project, Map<Str
     sonarQubeTask.dependsOn(getJavaCompileTasks(project));
     sonarQubeTask.mustRunAfter(getJacocoTasks(project));
     sonarQubeTask.dependsOn(getAndroidCompileTasks(project));
+    setNotCompatibleWithConfigurationCache(sonarQubeTask);
+  }
+
+  private static void setNotCompatibleWithConfigurationCache(SonarQubeTask sonarQubeTask) {
+    if (isGradleVersionGreaterThan("7.4.0")) {
+      sonarQubeTask.notCompatibleWithConfigurationCache("Plugin is not compatible with configuration cache");
+    }
+  }
+
+  private static boolean isGradleVersionGreaterThan(String version) {
+    return GradleVersion.current().compareTo(GradleVersion.version(version)) >=0;
```

Comment:
Looks like the comparison is "greater or equals" which doesn't match the method name?

Reply:
Done